lib/relation-definition.js

```
@@ -920,10 +920,14 @@ HasMany.prototype.updateById = function(fkId, data, options, cb) {
     options = {};
   }
   cb = cb || utils.createPromiseCallback();
+  var fk = this.definition.keyTo;
+
   this.findById(fkId, options, function(err, inst) {
     if (err) {
       return cb && cb(err);
     }
+    // Ensure Foreign Key cannot be changed!
+    if (fk) delete data[fk];
```

I wonder if we should report a 400 error or just silently remove the offending FK.

---

I was initially returning an error message but say that this was a case handled in the `BelongsTo` by silently dropping the FK before my changes. So I opted for that route as a result. I'm ok to switch them all to error messages if you think that's a better approach.